spec/spec_helper.rb

```diff
@@ -48,8 +48,11 @@
   # When we're testing the API, only run the api tests
   config.filter_run api: true if ENV['BLACKLIGHT_API_TEST']
 
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = Rails.root.join("spec/fixtures")
+  if Rails.version.to_f >= 7.1
+    config.fixture_paths = [Rails.root.join("spec/fixtures")]
+  else
+    config.fixture_path = Rails.root.join("spec/fixtures")
+  end
```

Comment:
Found this change in a commit in `main`. Without this change, the call to `#fixture_path=` raises an exception in Rails 8.